ror.yml

```diff
@@ -0,0 +1,147 @@
+# Recipe for Ruby on Rails collections
+---
+rh-ror50:
+  name: Ruby on Rails 5.0
+  requires: [rh-ruby23, rh-nodejs4, rh-mongodb32]
+  # install: Copr - Settings - Build options - Click "Edit" on target
+  #   Packages
+  install:
+    - scl-utils-build
+    - rh-ruby23-scldevel
+    - rh-ror50-build
+  packages:
+    - rh-ror50
+    # Packages required by RSpec, Cucumber
+    - rubygem-rspec
+    - rubygem-rspec-core:
+        replaced_macros:
+          need_bootstrap_set: 1
+    - rubygem-rspec-support:
+        replaced_macros:
+          need_bootstrap_set: 1
+    - rubygem-diff-lcs:
+        macros:
+          _with_bootstrap: 1
```

Comment:
Hello, this seems fine with current state, but IMHO there could be done a simplification:

Me and @voxik think that it would be better to have all the macros bootstrapping named all same. In current Fedora guidelines[1], there is 'bootstrap' and '_with_bootstrap' used.

Manually, you could then set the %bootstrap in spec file for the build, or we could introduce RPM package defining this macro(this needs the %tag)

Note that the %bootstrap is 1 when the system is already bootstrapped. I think this is counterintuitive and I will create a ticket with a request for the change(to flip the value). You can see how the decision was made here[2] and a possible solution[3] is mentioned in comment by @voxik. 

[1] https://fedoraproject.org/wiki/Packaging:Guidelines#Bootstrapping
[2] https://fedorahosted.org/fpc/ticket/469
[3] http://pkgs.fedoraproject.org/cgit/rpms/rubygem-sprockets-rails.git/commit/?id=3c57bf760f4cfc0dc692a503c51053aada74d284